rel/files/riak-repl

```diff
@@ -6,6 +6,7 @@
 ORIGINAL_DIR=$(pwd)
 
 # Make sure CWD is set to runner run dir
+RUNNER_BASE_DIR={{runner_base_dir}}
```

Comment:
Hey @hanssv, did something break that required this to be added? I think the runner_base stuff is overspill from the former node_package stuff, but isn't necessarily required any more, to my knowledge (needs a bit of cleanup to remove it).

Reply:
Yes, without this it runs in my home-directory, which means it does not find the SSL-config and thus can't talk to the node (or well it will refuse to even try to talk to it).

It wasn't necessary before (and it was horribly broken since bbmustache had HTML-encoded the bash command!)